indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -41,15 +41,38 @@
                     </button>
                 </div>
                 <div class="group">
-                    <a class="i-button icon-plus js-dialog-action hide-if-locked"
-                       title="{% trans %}Add new abstract{% endtrans %}"
-                       data-title="{% trans %}Add new abstract{% endtrans %}"
-                       data-href="{{ url_for('.manage_create_abstract', event) }}"
-                       data-update='{"html": "#abstract-list", "filter_statistics": "#filter-statistics"}'
-                       data-confirm-close-unsaved
-                       data-ajax-dialog>
-                        {% trans %}New abstract{% endtrans %}
+                    <a class="i-button arrow icon-plus hide-if-locked" data-toggle="dropdown">
+                        {% trans %}New{% endtrans %}
                     </a>
+                    <ul class="i-dropdown">
+                        <li>
+                            <a class="js-dialog-action"
+                               title="{% trans %}Add new abstract{% endtrans %}"
+                               data-title="{% trans %}Add new abstract{% endtrans %}"
+                               data-href="{{ url_for('.manage_create_abstract', event) }}"
+                               data-update='{"html": "#abstract-list", "filter_statistics": "#filter-statistics"}'
+                               data-confirm-close-unsaved
+                               data-ajax-dialog>
+                                {% trans %}Abstract{% endtrans %}
+                            </a>
+                        </li>
+                        <li>
+                            {%- if can_create_invited_abstracts -%}
+                                {%- set message = _("Add new invited abstract") %}
+                            {%- else -%}
+                                {%- set message = _("Define 'Invited' notification template in order to be able to create invited abstracts") %}
```

Comment:
"You first need to create ..."? The current text sounds like clicking the button would define this notification template.